lib/rouge/lexers/make.rb

```diff
@@ -28,15 +37,25 @@ def initialize(opts={})
 
         rule %r/#.*?\n/, Comment
 
-        rule %r/(export)(\s+)(?=[a-zA-Z0-9_\${}\t -]+\n)/ do
+        rule %r/([-s]?include)((?:[\t ]+[^\t\n #]+)+)/ do
+          groups Keyword, Literal::String::Other
+        end
+
+        rule %r/(ifn?def|ifn?eq)([\t ]+)([^#\n]+)/ do
+          groups Keyword, Text, Name::Variable
+        end
+
+        rule %r/(?:else|endif)[\t ]*(?=[#\n])/, Keyword
+
+        rule %r/(export)([\t ]+)(?=[a-zA-Z0-9_\${}()\t -]+\n)/ do
```

Comment:
I realise this is from the original source but thought it might be something we could fix while we're here:

```suggestion
        rule %r/(export)([\t ]+)(?=[\w${}()\t -]+\n)/ do
```

Reply:
OK, I've pushed another commit that does this (and the case below).